### PR TITLE
Add acceptance tests for the footer pages

### PR DESCRIPTION
**Pull request by brenetic**
This adds an acceptance tests for the footer pages when a new service is
created

### CODE REVIEW

File: acceptance/features/editing_standalone_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    editor.footer_pages_links.click
+  end
+
+  def and_I_edit_the_cookies_page
+    editor.cookies_link.click
+  end
+
+  def and_I_change_the_page_heading(heading)
+    editor.page_heading.set(heading)
+  end
```
Comment (tomas-stefano) [inline acceptance/features/editing_standalone_page_spec.rb:31]: I think this method already exists https://github.com/ministryofjustice/fb-editor/blob/main/acceptance/features/edit_confirmation_page_spec.rb#L35
If is the same behaviour, can we move to the common steps module?

**Review by tomas-stefano — APPROVED** [review]
🚀 Just a minor thing but everything look awesome

**Review by njseeto — DISMISSED** [review]
🚀

**Review by njseeto — APPROVED** [review]
🚀

**Pull request merged**